### PR TITLE
Issue #213 - EquinoxResolver slow to resolve

### PR DESCRIPTION
**Pull request by mickaelistria**

### CODE REVIEW

File: tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java [hunk 1]
```diff
@@ -202,6 +202,16 @@ protected ModuleContainer newState(DependencyArtifacts artifacts, Properties pro
             throws BundleException {
         ModuleContainer[] moduleContainerAccessor = new ModuleContainer[1];
         ModuleContainerAdaptor moduleContainerAdaptor = new ModuleContainerAdaptor() {
+
+            @Override
+            public String getProperty(String key) {
+                switch (key) {
+                case "equinox.resolver.revision.batch.size":
```
Comment (laeubi) [inline tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java:209]: Maybe we should add a comment /reference to bug to explain why this is necessary?

File: tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java [hunk 2]
```diff
@@ -202,6 +202,16 @@ protected ModuleContainer newState(DependencyArtifacts artifacts, Properties pro
...
+                    return Integer.toString(1);
```
Comment (laeubi) [inline tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java:210]: Maybe we could replace this by System.getProperty("tycho.resolver.batch.size", "1") to allow configuration through system properties.

**Comment by mickaelistria** [comment]
Merged as 47726f1745c7d8836a4ea02ba53ff4bb34b553c5

File: tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java (hunk 2 above)
Comment (mickaelistria) [reply tycho-core/src/main/java/org/eclipse/tycho/core/osgitools/EquinoxResolver.java:210]: I didn't add it. More options is more complexity and are only worth the effort when we know we need them.

**Pull request merged**